drivers/i2c/i2c_ll_stm32.c

```diff
@@ -53,9 +56,23 @@ int i2c_stm32_runtime_configure(const struct device *dev, uint32_t config)
 	data->dev_config = config;
 
 	k_sem_take(&data->bus_mutex, K_FOREVER);
+
+#ifdef CONFIG_PM_DEVICE_RUNTIME
+	(void)pm_device_runtime_get(dev);
+#else
+	pm_device_busy_set(dev);
+#endif
```

Comment:
@ceolin Is the the expected code when both PM configurations have to be supported ?